src/main/resources/com/google/jenkins/plugins/computeengine/InstanceConfiguration/config.jelly

```diff
@@ -44,6 +44,9 @@
             <f:entry field="preemptible" title="${%Preemptible?}">
                 <f:checkbox/>
             </f:entry>
+             <f:entry field="minCpuPlatform" title="${%Minimum Cpu Platform}">
```

Comment:
Can you add a help html file for minCpuPlatform? e.g. help-description.html, help-namePrefix.html

Reply:
Hmm thought I did maybe it didn't get added.

Reply:
dang